### PR TITLE
Fix config dir creation

### PR DESCRIPTION
**Pull request by samdobson**
Small fix for a regression with the config setup introduced as part of the whoosh implementation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ On first run you will be asked for some information so that the app can build yo
 
 You'll need to enter a valid Key Vault name.
 
-Alternatively you can manually pre-load a config file at `~/.azure-keyvault-browser.toml` and use the following schema:
+Alternatively you can manually pre-load a config file at `~/azure-keyvault-browser/config.toml` and use the following schema:
```
Comment (chelnak) [inline README.md]: The config lives in `.config` now

https://github.com/chelnak/azure-keyvault-browser/blob/81f914ee2c2eeb641350610a37af97d941b374eb/src/azure_keyvault_browser/config.py#L23

So i think this should be `~/.config/azure-keyvault-browser/config.toml`

**Review by chelnak — CHANGES_REQUESTED** [review]
Thanks again for the contribution! I've left a small comment for you to resolve then I will merge this 👍

File: README.md (hunk 1 above)
Comment (samdobson) [reply README.md]: Ah yes, of course. Good spot. Fixed in 24270ea6795ee14e6cc1039d30e64e3db18cd4ab.

**Comment by chelnak** [comment]
LGTM!

**Pull request merged**